dbt/include/athena/macros/materializations/seeds/helpers.sql

```diff
@@ -31,7 +31,54 @@
     ) as {{ col }}
 {% endmacro %}
 
-{% macro athena__create_csv_table(model, agate_table) %}
+{% macro create_csv_table_insert(model, agate_table) %}
+  {%- set identifier = model['alias'] -%}
+
+  {%- set lf_tags = config.get('lf_tags', default=none) -%}
+  {%- set lf_tags_columns = config.get('lf_tags_columns', default=none) -%}
```

Comment:
```suggestion
{%- set lf_tags_config = config.get('lf_tags_config') -%}
{%- set lf_grants = config.get('lf_grants') -%}
```
pls pay attention about updates in lakeformation integration